### ISSUE TITLE
Official Documentation On Wordpress Config

### ISSUE THREAD

**Issue by Fatman2142** [NginxProxyManager/nginx-proxy-manager#3813]
Hi Guys,

I have found tons on the internet around Nginx and Wordpress but nothing on Nginx Proxy Manager and Wordpress. Have been troubleshooting my install for a few weeks with no success. 

Would it be possible for the official documentation on how to route WordPress sites? 

Very least it will help with preventing bug reports for those who don't realize all the things they need to do. 

Thanks!

**Comment by remylavergne** [comment]
Did you find a solution?

**Comment by thessy** [comment]
I agree to @Fatman2142 and would love to see such an official documentation on 'docker, wordpress and with nginx-proxy-manager'

The same in https://github.com/NginxProxyManager/nginx-proxy-manager/issues/400 A lot of hints, examples, but most of them about 1-4 years old. Nothing worked for me... I'm getting '502 Bad Gateway', WP over http:// is running fine.

**Comment by RemyDG0** [comment]
I tried to set up nginx-proxy-manager for my WordPress site but the web page is responding a 502 bad gateway error. 
Here are some Screenshot. 
I just can't make it work. Can anyone help me please ?
![image](https://github.com/user-attachments/assets/414b10db-f585-4f37-8ae8-0cab2a46491b)

<img width="452" alt="image" src="https://github.com/user-attachments/assets/8fb06e7b-0c7f-4d4c-824e-ba1bc095e5da">
<img width="455" alt="image" src="https://github.com/user-attachments/assets/7d9065fc-bd0f-4339-bf5f-3bae01def669">

**Comment by RemyDG0** [comment]
@remylavergne I still do. 
My problem is I don't really know where to look, all tutorials I saw only talked about command lines or files edit, it is just like if the web gui does not exists. Then I am not really sure how to access those.

**Comment by remylavergne** [comment]
I can maybe help you if you have same "config" as mine : 
- using a VPS
- nginx proxy manager deployed with docker
- wordpress deployed with docker

Let me know.

**Comment by RemyDG0** [comment]
@remylavergne 
Did you mean a VPN instead of VPS ? 
I don't use either. 
I use both packages as docker images but the docker host is TrueNas scale and they customised it a little. 
Does the configuration need to be done outside of of the Web user interface ?

**Comment by remylavergne** [comment]
I don't have expertise with _TrueNAS_ sorry :/

**Comment by RemyDG0** [comment]
> I don't have expertise with _TrueNAS_ sorry :/

I found how to access the console, the docker image and web gui are the same, my question if settings for setting up Wordpress are only available in the console

**Comment by remylavergne** [comment]
My NPM config : 
![image](https://github.com/user-attachments/assets/e8c38b29-9750-4646-a0f2-b1d0bd10b6bb)
The hostname / ip should point to the Wordpress container.

In Wordpress you should add the doamin in configuration : 
![image](https://github.com/user-attachments/assets/8c606d3d-4db4-4090-8b0f-699382f61968)

Don't forget to add your wordpress container in the same network as NPM (my biggest mistake) : 

```yaml
networks:
  proxy_manager_default:
    external: true
```

The `docker ps` output : 
![image](https://github.com/user-attachments/assets/24e6eb91-eabe-420b-accc-8e10c497e998)

I assume that your TrueNAS runs inside a private network, so don't forget to open internet box ports.

**Comment by RemyDG0** [comment]
Thank you so much for your config @remylavergne, I now understand what was not working for me ! 
My mistake was basically to select **https** instead of **http** for the _scheme_ in the _Edit Proxy Host_ window. 
It mislead me making me think that it was necessary in order to use ssl.

**Comment by remylavergne** [comment]
@RemyDG0 Perfect 😎

**Comment by scyto** [comment]
I have had a working wordpress install

it no longer works correctly behined NPM

I am getting this error:

```
Bad Request
Your browser sent a request that this server could not understand.

Apache/2.4.62 (Debian) Server at 10.0.16.18 Port 80
```

Initially i thought this was a corruption issue with my wordpress install, however i have hit exactly the same issue with a brandnew install

I have used the following custom location settings for /

```
proxy_set_header Host $host:$server_port;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forwarded-Proto $scheme;
```


any ideas?

**Comment by remylavergne** [comment]
Maybe try to not force domain through WP config and let proxy managing it.

```
        define('DOMAIN_CURRENT_SITE', 'mydomain.com'); 
```
